app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -65,11 +65,12 @@
     "update": "Update",
     "cancel": "Cancel",
     "eventSubscription": "Event Subscription",
-    "cdogsTemplate": "CDOGS Template"
+    "cdogsTemplate": "CDOGS Template",
+    "externalAPIs": "External APIs"
```

Comment:
Probably goes without saying, but we'll also need the translations for the other languages.

Reply:
I've added a new ticket in the backlog to update i18n.